RIS.js

```diff
@@ -1395,9 +1395,16 @@ function processTag(item, tagValue, risEntry, allowDeprecated) {
 					
 					//get title from file name
 					title = url.match(/([^\/\\]+)(?:\.\w{1,8})$/);
-					if (title) title = decodeURIComponent(title[1]);
-					else title = "Attachment";
-					
+					if (title) {
+						try {
+							title = decodeURIComponent(title[1]);
+						}
+						catch (e) {}
```

Comment:
I initially misread this and thought that an error would result in an empty title, but it looks like the issue is actually that an error will result in gibberish, since it'll set the attachment title to the regex match array. Try this?

```suggestion
						catch (e) {
							title = title[1];
						}
```